controllers/nginx/configuration.md

```diff
@@ -110,49 +110,47 @@ Indicates the [HTTP Authentication Type: Basic or Digest Access Authentication](
+The secret must be created in the same namespace as the Ingress rule.
 
 ```
 ingress.kubernetes.io/auth-realm:"realm string"
 ```
 
-Please check the [auth](examples/auth/README.md) example
+Please check the [auth](examples/auth/README.md) example.
 
 
 ### External Authentication
 
 To use an existing service that provides authentication the Ingress rule can be annotated with `ingress.kubernetes.io/auth-url` to indicate the URL where the HTTP request should be sent.
-Additionally is possible to set `ingress.kubernetes.io/auth-method` to specify the HTTP method to use (GET or POST) and `ingress.kubernetes.io/auth-send-body` to true or false (default).
+Additionally it is possible to set `ingress.kubernetes.io/auth-method` to specify the HTTP method to use (GET or POST) and `ingress.kubernetes.io/auth-send-body` to true or false (default).
 
 ```
 ingress.kubernetes.io/auth-url:"URL to the authentication service"
 ```
 
-Please check the [external-auth](examples/external-auth/README.md) example
+Please check the [external-auth](examples/external-auth/README.md) example.
 
 
 ### Rewrite
 
 In some scenarios the exposed URL in the backend service differs from the specified path in the Ingress rule. Without a rewrite any request will return 404.
 Set the annotation `ingress.kubernetes.io/rewrite-target` to the path expected by the service.
 
-If the application contains relative links is possible to add an additional annotation `ingress.kubernetes.io/add-base-url` that will append a `base` tag in the header of the returned HTML from the backend.
-
+If the application contains relative links it is possible to add an additional annotation `ingress.kubernetes.io/add-base-url` that will prepend a [`base` tag](https://developer.mozilla.org/en/docs/Web/HTML/Element/base) in the header of the returned HTML from the backend.
 
-Please check the [rewrite](examples/rewrite/README.md) example
+Please check the [rewrite](examples/rewrite/README.md) example.
 
 
 ### Rate limiting
 
-The annotations `ingress.kubernetes.io/limit-connections` and `ingress.kubernetes.io/limit-rps` allows the creation of a limit in the connections that can be opened by a single client IP address. This can be use to mitigate [DDoS Attacks](https://www.nginx.com/blog/mitigating-ddos-attacks-with-nginx-and-nginx-plus)
-
-`ingress.kubernetes.io/limit-connections`: number of concurrent allowed connections from a single IP address
+The annotations `ingress.kubernetes.io/limit-connections` and `ingress.kubernetes.io/limit-rps` define a limit on the connections that can be opened by a single client IP address. This can be used to mitigate [DDoS Attacks](https://www.nginx.com/blog/mitigating-ddos-attacks-with-nginx-and-nginx-plus).
 
-`ingress.kubernetes.io/limit-rps`: number of allowed connections per second from a single IP address
+`ingress.kubernetes.io/limit-connections`: number of concurrent allowed connections from a single IP address.
```

Comment:
I think "number of allowed concurrent connections" or "number of concurrent connections allowed".